eleventy.config.js

```diff
@@ -20,6 +23,7 @@ module.exports = function (eleventyConfig) {
     url:
       process.env.GITHUB_ACTIONS &&
       'https://x-govuk.github.io/govuk-eleventy-plugin/',
+    feedUrl: 'https://x-govuk.github.io/govuk-eleventy-plugin/feed.xml',
     stylesheets: ['/assets/application.css'],
```

Comment:
Not sure the documentation site needs a feed, and the feed doesn’t include anything, anyway.

```suggestion
      'https://x-govuk.github.io/govuk-eleventy-plugin/',
    stylesheets: ['/assets/application.css'],
```

Reply:
Agree. 👍